### ISSUE TITLE
onnx_to_trt

### ISSUE THREAD

**Issue by xulinjie522** [TUI-NICR/ESANet#59]
Hello, how do I convert onnx to TRT, can you provide the relevant conversion code? When I get the TRT model, how do I use it?

**Comment by danielS91** [comment]
Yes, see [model_to_onnx.py](https://github.com/TUI-NICR/ESANet/blob/main/model_to_onnx.py) and [inference_time_whole_model.py](https://github.com/TUI-NICR/ESANet/blob/main/inference_time_whole_model.py).

However, as the code is already quite old, I would recommend to have a look at the scripts in our follow-up works [EMSAFormer](https://github.com/TUI-NICR/EMSAFormer) and [EMSANet](https://github.com/TUI-NICR/EMSANet).
